natives.json

```diff
@@ -110573,11 +110573,11 @@
 					"name": "p9"
 				},
 				{
-					"type": "Any",
-					"name": "p10"
+					"type": "char*",
```

Comment:
I use proper ``const char*`` arguments, so please change it to that.
Rest looks good 👍